arviz/plots/ppplot.py

```diff
@@ -0,0 +1,157 @@
+    coords : dict
+        Dictionary mapping dimensions to selected coordinates to be plotted.
+        Dimensions without a mapping specified will include all coordinates for
+        that dimension.
+    transform : callable
+        Function to transform data (defaults to None i.e. the identity function)
+    legend : bool
+        Add legend to figure. By default True.
+    ax: axes, optional
+        Matplotlib axes: a numpy 2d array of matplotlib axes. Returned object will
+        have shape (nvars, 3) where the last column is the combined plot and the
+        first columns are the single plots.
+    prior_kwargs : dicts, optional
+        Additional keywords passed to `arviz.plot_kde` for prior group.
+    posterior_kwargs : dicts, optional
+        Additional keywords passed to `arviz.plot_kde` for posterior group.
+    backend: str, optional
+        Select plotting backend {"matplotlib","bokeh"}. Default "matplotlib".
+    backend_kwargs: bool, optional
+        These are kwargs specific to the backend being used. For additional documentation
+        check the plotting method of the backend.
+    show : bool, optional
+        Call backend show function.
+
+    Returns
+    -------
+    axes : matplotlib axes or bokeh figures
+
+    Examples
+    --------
+    Plot the prior/posterior plot for specified vars and coords.
+
+    .. plot::
+        :context: close-figs
+
+        >>> import arviz as az
+        >>> data = az.load_arviz_data('radon')
+        >>> az.plot_pp(data, var_names=["defs"], coords={"team" : ["Italy"]})
+
+    """
+
+    groups = ["prior", "posterior"]
+
+    if coords is None:
+        coords = {}
+
+    if prior_kwargs is None:
+        prior_kwargs = {}
+
+    if posterior_kwargs is None:
+        posterior_kwargs = {}
+
+    if backend_kwargs is None:
+        backend_kwargs = {}
+
+    datasets = [getattr(data, group) for group in groups]
+
+    if var_names is None:
+        var_names = list(datasets[0].data_vars)
+
+    if isinstance(var_names, str):
+        var_names = [var_names]
+
+    if isinstance(var_names[0], str):
+        var_names = [var_names for _ in datasets]
+
+    if transform is not None:
+        datasets = [transform(dataset) for dataset in datasets]
+
+    datasets = get_coords(
+        datasets, list(reversed(coords)) if isinstance(coords, (list, tuple)) else coords
+    )
+    pp_plotters = [
+        list(xarray_var_iter(data, var_names=var, combined=True))
+        for data, var in zip(datasets, var_names)
+    ]
+
+    nvars = len(pp_plotters[0])
+    ngroups = len(groups)
+
+    (figsize, ax_labelsize, _, xt_labelsize, linewidth, markersize) = _scale_fig_size(
+        figsize, textsize, 2 * nvars, ngroups
+    )
+
+    posterior_plot_kwargs = posterior_kwargs.pop("plot_kwargs", dict())
+    posterior_plot_kwargs.setdefault("color", "red")
+    posterior_plot_kwargs.setdefault("linewidth", linewidth)
+    posterior_kwargs["plot_kwargs"] = posterior_plot_kwargs
```

Comment:
I think there is no need to pop, `posterior_plot_kwargs["plot_kwargs"].setdefault(...)` should work